bridge/e2e/terra.go

```diff
@@ -166,9 +170,18 @@ func getAssetAddress(ctx context.Context, contract string, chain uint8, asset []
 	return gjson.Get(json, "result.address").String(), nil
 }
 
+func padAddress(address common.Address) vaa.Address {
```

Comment:
We now have three copies of this, time to refactor into common package with some documentation what it does and replace all of them.